fe/fe-core/src/main/java/com/starrocks/catalog/HiveTable.java

```diff
@@ -322,16 +322,17 @@ private void validate(Map<String, String> properties) throws DdlException {
         copiedProps.remove(HIVE_TABLE);
 
         // check hive properties
-        // hive.metastore.uris or hive.resource must be set
+        // resource must be set and hive.metastore.uris will be ignored if specified.
         String hiveMetastoreUris = copiedProps.get(HIVE_METASTORE_URIS);
         String resourceName = copiedProps.get(HIVE_RESOURCE);
-        if (Strings.isNullOrEmpty(hiveMetastoreUris) && Strings.isNullOrEmpty(resourceName)) {
-            throw new DdlException("property " + HIVE_METASTORE_URIS + " or " + HIVE_RESOURCE + " must be set");
+        if (Strings.isNullOrEmpty(resourceName)) {
+            throw new DdlException("property " + HIVE_RESOURCE + " must be set");
         }
 
         if (!Strings.isNullOrEmpty(hiveMetastoreUris)) {
             copiedProps.remove(HIVE_METASTORE_URIS);
-            hiveProperties.put(HIVE_METASTORE_URIS, hiveMetastoreUris);
+            LOG.warn("property " + HIVE_METASTORE_URIS + " will be ignored " +
+                    "and hive table will be created by using property " + HIVE_RESOURCE + " only.");
         }
 
         if (!Strings.isNullOrEmpty(resourceName)) {
```

Comment:
Remove this check because it has already been guaranteed.

Reply:
done, PTAL